pootle/settings/30-site.conf

```diff
@@ -15,6 +15,8 @@ MANAGERS = ADMINS
 
 DEBUG = False
 
+CANONICAL_URL = ""
```

Comment:
@phlax yeah I get that, but now we can init/initdb without much effort, this would change that.  I think missing should fail, and yip if we just upgrade 'localhost' to a warning that would cover things I think.